rust_src/remacs-sys/lib.rs

```diff
@@ -973,6 +973,7 @@ extern "C" {
     pub static Qoverlayp: Lisp_Object;
     pub static Qminus: Lisp_Object;
     pub static Qmark_inactive: Lisp_Object;
+    pub static Qunbound: Lisp_Object;
```

Comment:
The place I inserted these values was somewhat arbitrary. Is there a protocol for adding FFI bits?

Reply:
not yet.

Reply:
Hence my post about style guidelines.